client/components/pickup/PickupCarousel.tsx

```diff
@@ -47,9 +46,43 @@ const PickupCarousel = () => {
 				}}
 			>
 				{slides.map((slide, index) => (
-					<div key={index}>
-						<img src={slide.url} alt={slide.url} />
-					</div>
+					<Box key={index} minWidth={"100%"} style={{ position: "relative" }}>
+						<Image src={slide.url} alt={slide.url} />
+						<div
+							style={{
```

Comment:
can try to make this style more readable? maybe define in not inline (create a const for it), and if possible reduce the nested ternarys